atm.js

```diff
@@ -1,21 +1,123 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//select / create elements to work with
+console.log('ready');
+var checkingBox = $('#check-input');
```

Comment:
I would consider either putting these global variables into another object, adding them as properties on your checkingAcct and/or savingsAcct objects ```checkingAcct.checkingBox = $('#check-input')``` or on the prototype of Acct constructor ```Acct.prototype.checkingbox = $("#check-input")```